pegasus/sites.v3/code.org/public/mc.haml

```diff
@@ -22,34 +22,31 @@ social:
 -# Because of partner requirements, all urls need to use mc, but our internal shortcut is "mchoc".
 -# Hoc.com has a mc.redirect to his page and the router has a check for "code.org/MC" and "hoc.com/MC"
 
-- page_mode = PageMode.get(request)
-
```

Comment:
We only had it on the /mc page because we wanted it to be hidden until we flipped the switch. We can just show everything now, doesn't matter what the page mode is here.